### PR TITLE
Add stats to jwtkmanager

### PR DESCRIPTION
**Pull request by skiptomyliu**
Allows us to obtain metrics on `get_jwt`  function so we graph latency on jwt encoding, jwt cache hits and misses.

### CODE REVIEW

File: confidant/services/jwkmanager.py [hunk 1]
```diff
@@ -67,17 +69,20 @@ def get_jwt(self, kid: str, payload: dict,
...
         })
 
-        token = jwt.encode(
-            payload=payload,
-            headers={'kid': kid},
-            key=key.export_to_pem(private_key=True, password=None),
-            algorithm=algorithm,
-        )
+        with stats.timer('get_jwt.encode'):
+            # XXX: TODO: cache export_to_pem
```
Comment (skiptomyliu) [inline confidant/services/jwkmanager.py:73]: @erickduran , keeping track of this, we may be able to speed this function up by storing the pem in memory.

**Review by erickduran — APPROVED** [review]

File: confidant/services/jwkmanager.py [hunk 2]
```diff
@@ -67,17 +69,20 @@ def get_jwt(self, kid: str, payload: dict,
...
+                headers={'kid': kid},
+                key=key.export_to_pem(private_key=True, password=None),
+                algorithm=algorithm,
+            )
 
         self._token_cache[kid][user] = {
             'expiry': expiry,
             'token': token
         }
+        stats.incr('get_jwt.cache.miss')
```
Comment (erickduran) [inline confidant/services/jwkmanager.py:85]: do we want to also record this is caching is disabled?

Comment (skiptomyliu) [reply confidant/services/jwkmanager.py]: hrmm, yeah that's a good point.  i've renamed this to `get_jwt.create`

**Comment by skiptomyliu** [comment]
@erickduran sorry, need another review after dismissing due to updating the PR.

**Review by erickduran — APPROVED** [review]

**Pull request merged**